src/server/graphql/models/Task/rebalanceTask.js

```diff
@@ -0,0 +1,15 @@
+import r from 'server/database/rethinkDriver';
+import {STEP} from 'universal/utils/constants';
```

Comment:
good change.
I ended up using floats because `GraphQLInt` is a signed 32-bit int 😑 .